userguide/tutorials/payment_plugin.adoc

```diff
@@ -1,44 +1,37 @@
 = Payment Plugin Overview
 
-Payment plugins integrate Kill Bill with a specific gateway (i.e. payment processor), such as Stripe or Braintree. We already have many open-source payment plugins available on GitHub as follows:
+Payment plugins integrate Kill Bill with a specific https://docs.killbill.io/latest/Kill-Bill-Glossary.html#payment_gateway[gateway] (i.e. payment processor), such as Stripe or Braintree. We already have many open-source payment plugins available on GitHub as listed in our https://docs.killbill.io/latest/plugin_introduction.html#payment_gateways[_plugin document_].
 
-. https://github.com/killbill/killbill-stripe-plugin[_killbill-stripe-plugin_] - Plugin to use  https://stripe.com/[_Stripe_] as the payment gateway
+This tutorial can be used to develop a custom payment plugin.
```

Comment:
To rephrase in active voice, "You can use this tutorial to develop a customer payment plugin." or "Use this tutorial...".